src/Command/AssertBackwardsCompatible.php

```diff
@@ -148,7 +148,7 @@ public function execute(InputInterface $input, OutputInterface $output) : int
         // @todo fix flaky assumption about the path of the source repo...
         $sourceRepo = CheckedOutRepository::fromPath(getcwd());
 
-        $fromRevision = $input->hasOption('from')
+        $fromRevision = $input->getOption('from') !== null
```

Comment:
This kind of change requires a test addition

Reply:
Yep sorry, didn't see the command was tested. Will add.

Reply:
test already exists, updated it.